aaelf64/aaelf64.rst

```diff
@@ -1780,7 +1778,7 @@ Relocations ``R_AARCH64_TLS_DTPREL``, ``R_AARCH64_TLS_DTPMOD`` and ``R_AARCH64_T
 
 It is implementation defined whether ``R_<CLS>_TLS_IMPDEF1`` implements ``R_<CLS>_TLS_DTPREL`` and ``R_<CLS>_TLS_IMPDEF2`` implements ``R_<CLS>_TLS_DTPMOD`` or whether ``R_<CLS>_TLS_IMPDEF1`` implements ``R_<CLS>_TLS_DTPMOD`` and ``R_<CLS>_TLS_IMPDEF2`` implements ``R_<CLS>_TLS_DTPREL``; a platform must document its choice\ [#aaelf64-f1]_.
 
-``R\_<CLS\_AUTH\_ABS64``, ``R\_<CLS>\_AUTH\_ABS64`` and ``R\_<CLS>\_AUTH\_RELATIVE`` are part of the PAuth ABI Extension. For details on the relocations and operations see `PAUTHABIELF64`_.
+``R\_<CLS>\_AUTH\_ABS64`` and ``R\_<CLS>\_AUTH\_RELATIVE`` are part of the PAuth ABI Extension. For details on the relocations and operations see `PAUTHABIELF64`_. Note that for their respective file format ``R\_<CLS>\_AUTH\_ABS64`` is both a static and a dynamic relocation.
```

Comment:
I find this somewhat confusingly phrased. The way you described it in the comments is a lot clearer:
`Like R_<CLS>_ABS64, R_<CLS>_AUTH_ABS64 shares the same relocation code in both static and dynamic linking.` Would that work?

Or otherwise rephrase it. The `their` refers to both the file formats, but they are defined in the same sentence in a way that doesn't sound correct to me, and I read the `their` as relating to ``R\_<CLS>\_AUTH\_ABS64``.

Reply:
I chose the wording to stay consistent with what we have for `R_<CLS>_ABS64` https://github.com/ARM-software/abi-aa/blob/main/aaelf64/aaelf64.rst?plain=1#L1755 which is what `R_<CLS>_AUTH_ABS64` is modelled on.

I don't have to use the same wording though. I'll try rephrasing.

Reply:
The earlier text is:
```
``R_<CLS>_ABS64`` and ``R_<CLS>_ABS32`` may only appear in a well-formed executable or dynamic shared object in ELF64 or ELF32 respectively. Note that for their respective file format these relocations are both static and dynamic relocations.
```
As there are two relocations, with two file formats ELF64 and ELF32 the respective file format makes sense.

I think I can just drop that part and retain the language:
```
Note that ``R\_<CLS>\_AUTH\_ABS64`` is both a static and a dynamic relocation.
```

Reply:
Yup that works. Thanks!